test/test_basic.rb

```diff
@@ -6,6 +6,8 @@
 
 class TestChangelogerator < Test::Unit::TestCase
   def test_polkadot_1_commit
+    return unless ENV['GITHUB_TOKEN']
+
```

Comment:
I don't think this is a good idea since it would give the false impression that everything is OK. Those tests make no sense without a `GITHUB_TOKEN` and the tests should FAIL and not PASS if the token is not set.

```suggestion
    raise "Missing GITHUB_TOKEN token" unless ENV["GITHUB_TOKEN"]

```

Reply:
Ok, implemented
https://github.com/paritytech/changelogerator/actions/runs/4173264328/jobs/7225385346#step:6:9